datastore-api/tests/test_kgs.py

```diff
@@ -104,7 +150,81 @@ def test_delete_kg(self, client: TestClient, token: str, conceptnet_kg):
         assert response_get.status_code == expected_code_get_sucess     
         assert response_delete.status_code == expected_code_delete
         assert response_get2.status_code == expected_code_get_not_found
-    
+
+    #################################
+    # WIKIDATA TESTS
+    def test_wikidata_alive(self, client: TestClient, wikidata_kg_name: str):
+        # Given:
+        url = f"/datastores/kg/{wikidata_kg_name}"
+
+        # When:
+        response = client.get(url)
+
+        # Then:
+        assert response.json() == 'Wikidata API is alive'
+
+    def test_wikidata_get_nodes_by_names(self, client: TestClient, wikidata_kg_name:str, wikidata_expected_nodes: dict):
+        # Given:
+        url = f"/datastores/kg/{wikidata_kg_name}/nodes/query_by_name"
+        expeceted_code = 200
+
+        # When:
+        response = client.post(
+            url,
+            json = ["Barack Obama", "Bill Clinton"] )
+        
+        # Then:
+        assert response.status_code == expeceted_code
+        assert response.json() == wikidata_expected_nodes
+
+
+    def test_wikidata_get_node_by_id(self, client: TestClient, wikidata_kg_name: str):
+        # Given:
+        node_id = "Q42"
+        url = f"/datastores/kg/{wikidata_kg_name}/{node_id}"
```

Comment:
Do we have such API in routers/kgs.py https://github.com/UKP-SQuARE/square-core/blob/master/datastore-api/app/routers/kgs.py?